scripts/build-docs.sh

```diff
@@ -21,12 +21,10 @@ appledoc \
     --no-warn-invalid-crossref \
     --no-warn-undocumented-object \
     --no-warn-undocumented-member \
-    --ignore "Realm/RLMRealm_Dynamic.h" \
-    --ignore "Realm/RLMArrayAccessor.h" \
-    --ignore "Realm/RLMArrayAccessor.mm" \
-    --ignore "Realm/RLMQueryUtil.h" \
-    --ignore "Realm/RLMUtil.h" \
-    --ignore "Realm/RLMRealm_Dynamic.h" \
+    --ignore ".m" \
```

Comment:
Ok, well do you want me to add the files that are in repo but shouldn't generate docs, even though they don't generate docs because the comment syntax isn't appledoc compatible?